src/osd/PrimaryLogPG.cc

```diff
@@ -728,7 +728,7 @@ void PrimaryLogPG::maybe_force_recovery()
   // find the oldest missing object
   version_t min_version = pg_log.get_log().head.version;
   hobject_t soid;
-  if (!pg_log.get_missing().get_items().empty()) {
+  if (!pg_log.get_missing().get_rmissing().empty()) {
```

Comment:
> rmissing != missing in backfill

why?

Reply:
in prep_backfill_object_push func, we know the needed version of all backfilling object is zero

Reply:
how is this connected to the conclusion of `missing.empty() != rmissing.empty()`?

Reply:
because we use rmissing iterator instead of missing iterator, rmissing.empty() looks more correctly

Reply:
so, i think it's more of a cleanup? am i right?

Reply:
yes

Reply:
could you update the commit message accordingly?